app/backend/gwells/management/commands/export_databc.py

```diff
@@ -0,0 +1,304 @@
+            target=target, timestamp=datetime.now() - start))
+
+    def write_geojson_header(self, f):
+        """Our header is always the same, a feature collection."""
+        f.write("""{
+    "type": "FeatureCollection",
+    "features": [
+""")
+
+    def generate_lithology(self, target):
+        # IMPORTANT: If the underlying data structure changes (e.g. column name changes etc.), the
+        # property names have to stay the same! This endpoint is consumed by DataBC and must remain
+        # stable!
+        sql = ("""
+select
+    ST_AsGeoJSON(geom) :: json as "geometry",
+    well.well_tag_number,
+    identification_plate_number,
+    well_status_code.description as well_status_description,
+    licenced_status_code.description as licenced_status_description,
+    CONCAT('https://apps.nrs.gov.bc.ca/gwells/well/',well.well_tag_number) as detail,
+    lithology_description.lithology_from,
+    lithology_description.lithology_to,
+    lithology_colour_code.description as lithology_colour_description,
+    lithology_description_code.description as lithology_description,
+    lithology_material_code.description as lithology_material_description,
+    lithology_observation,
+    lithology_hardness_code.description as lithology_hardness_description,
+    well_class_code.description as well_class_description,
+    intended_water_use_code.description as intended_water_use_description,
+    street_address,
+    finished_well_depth,
+    diameter,
+    static_water_level,
+    bedrock_depth,
+    well_yield,
+    well_yield_unit_code.description as well_yield_unit,
+    aquifer_id
+from well
+    inner join lithology_description on
+        lithology_description.well_tag_number = well.well_tag_number
+    left join well_status_code on
+        well_status_code.well_status_code = well.well_status_code
+    left join licenced_status_code on
+        licenced_status_code.licenced_status_code = well.licenced_status_code
+    left join lithology_material_code on
+        lithology_material_code.lithology_material_code =
+            lithology_description.lithology_material_code
+    left join lithology_colour_code on
+        lithology_colour_code.lithology_colour_code = lithology_description.lithology_colour_code
+    left join lithology_description_code on
+        lithology_description_code.lithology_description_code =
+            lithology_description.lithology_description_code
+    left join lithology_hardness_code on
+        lithology_hardness_code.lithology_hardness_code =
+            lithology_description.lithology_hardness_code
+    left join well_class_code on well_class_code.well_class_code = well.well_class_code
+    left join intended_water_use_code on
+        intended_water_use_code.intended_water_use_code = well.intended_water_use_code
+    left join well_yield_unit_code on
+        well_yield_unit_code.well_yield_unit_code = well.well_yield_unit_code
+    where well.well_tag_number >= {} and well.well_tag_number < {}
```

Comment:
the `{}` should both be replaced by `%s` https://docs.djangoproject.com/en/2.1/topics/db/sql/#executing-custom-sql-directly